workloads/devstack-xenial.yaml

```diff
@@ -57,14 +51,43 @@ runcmd:
     "DATABASE_PASSWORD=\$ADMIN_PASSWORD"
     "RABBIT_PASSWORD=\$ADMIN_PASSWORD"
     "SERVICE_PASSWORD=\$ADMIN_PASSWORD"
-    > {{template "DEVSTACKPATH" .}}/local.conf' {{.User}}
+    > /home/{{.User}}/devstack/local.conf' {{.User}}
   - {{endTaskCheck .}}
-{{ end }}
 
   - {{beginTask . "Stacking the system" }}
-  - su -c "cd {{template "DEVSTACKPATH" .}} && nohup ./stack.sh > /home/{{.User}}/stack.log &" {{.User}}
+  - su -c "cd /home/{{.User}}/devstack && nohup ./stack.sh > /home/{{.User}}/stack.log &" {{.User}}
```

Comment:
if we do not plan to start stacking by default, we should get rid of this task.

Reply:
Fantastic.  So this is why I was getting all of these errors from stack.sh.  I was running two stack.sh at the same time.

Reply:
I thought I'd removed it but obviously I didn't.  I'll get rid of it now.

Reply:
Aha!! that explains the race condition. :)